src/transformState.js

```diff
@@ -62,7 +62,19 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const modifier of transforms) {
+    if (modifier['operation'] === 'addProperties') {
```

Comment:
let's use switch case to make this code more readable

Reply:
ok